src/graph.cc

```diff
@@ -174,11 +174,22 @@ bool DependencyScan::RecomputeOutputDirty(Edge* edge,
     }
 
     if (output_mtime < most_recent_input->mtime()) {
+      time_t output_t = static_cast<time_t>(output_mtime);
+      time_t most_recent_input_t = static_cast<time_t>(most_recent_input->mtime());
+
+      char *output_t_buff = ctime(&output_t);
+      char *most_recent_input_t_buff = ctime(&most_recent_input_t);
+
+      char output_t_str[100] = {0,};
+      char most_recent_input_t_str[100] = {0,};
+      strncpy(output_t_str, output_t_buff, strlen(output_t_buff) - 1);
```

Comment:
Sorry for late comment. I did re-write the codes and make a commit.

I will request with the modificated version again.

Thanks